geometry_msgs/Point[] trianglePoints